src/main/java/jp/ikedam/jenkins/plugins/extensible_choice_parameter/ExtensibleChoiceParameterDefinition.java

```diff
@@ -121,12 +188,13 @@ public ExtensibleChoiceParameterDefinition newInstance(StaplerRequest req,
             if (formData == null || formData.isNullObject()) {
                 return null;
             }
-            if (!formData.has("stapler-class")) {
-                throw new FormException("No stapler-class is specified", fieldName);
+            String staplerClazzName = formData.optString("$class", null);
+            if (staplerClazzName == null) {
+                // Fall back on the legacy stapler-class attribute.
+                staplerClazzName = formData.optString("stapler-class", null);

```

Comment:
Changes for [JENKINS-25403](https://issues.jenkins-ci.org/browse/JENKINS-25403).